Add more lvgl arc update parameters (#5667)
OttoWinter/esphomedocs: content/components/lvgl/widgets.md

- **arc_opa** (*Optional*, [opacity](/components/lvgl#lvgl-opacity)): Opacity of the arc.
- **arc_rounded** (*Optional*, boolean): Make the end points of the arcs rounded. `true` rounded, `false` perpendicular line ending.
- **arc_width** (*Optional*, int16): Set the width of the arcs in pixels.

> - **change_rate** (*Optional*, uint16): If the arc is pressed the current value will set with a limited speed according to the set change rate. The change rate is defined in degree/second. Defaults to `720`.

Reviewer: ```suggestion
- **change_rate** (*Optional*, uint16): Limits the speed at which the arc value changes when touched or dragged. The change rate is defined in degree/second. Defaults to `720`.
```
